test/BreakInvariantBounty.test.js

```diff
@@ -17,13 +18,13 @@ contract('BreakInvariantBounty', function ([_, owner, researcher, anyone, nonTar
   });
 
   it('can set reward', async function () {
-    await ethSendTransaction({ from: owner, to: this.bounty.address, value: reward });
+    await sendEther(owner, this.bounty.address, reward);
```

Comment:
The `SplitPayment` tests also used `ethSendTransaction`, could you update those as well?